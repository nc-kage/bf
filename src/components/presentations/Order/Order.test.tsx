import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import Comp from '.';

describe('App', () => {
  it('snapshot left', () => {
    const { asFragment } = render(
      <Comp
        className="test"
        order={{ price: 1, amount: 1, count: 1 }}
        orientation="left"
      />,
    );
    expect(asFragment()).toMatchSnapshot();
  });

  it('snapshot right', () => {
    const { asFragment } = render(
      <Comp
        className="test"
        order={{ price: 1, amount: 1, count: 1 }}
        orientation="right"
      />,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
