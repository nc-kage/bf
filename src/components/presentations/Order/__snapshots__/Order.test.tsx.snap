// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot left 1`] = `
<DocumentFragment>
  <div
    class="Order Order--left test"
  >
    <span
      class="Order__item"
    >
      1
    </span>
    <span
      class="Order__item"
    >
      1
    </span>
    <span
      class="Order__item"
    >
      1
    </span>
  </div>
</DocumentFragment>
`;

exports[`App snapshot right 1`] = `
<DocumentFragment>
  <div
    class="Order Order--right test"
  >
    <span
      class="Order__item"
    >
      1
    </span>
    <span
      class="Order__item"
    >
      1
    </span>
    <span
      class="Order__item"
    >
      1
    </span>
  </div>
</DocumentFragment>
`;
